src/index.html

```diff
@@ -7,8 +7,31 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+      crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="https://www.moyo.ua/" class="logo"><img src="/moyo_1.svg" alt="logo"></a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item"><a href="https://www.moyo.ua/apple" class="nav__apple nav__link is-active">Apple</a></li>
+          <li class="nav__item"><a href="https://www.moyo.ua/samsung" class="nav__samsung nav__link">Samsung</a></li>
```

Comment:
- And fix formatting, don't put nested elements in the parent line
- Add empty lines between multiline sibling blocks of HTML
![image](https://github.com/mate-academy/layout_moyo-header/assets/63243136/211fb52c-7b85-4ab0-9638-cba3fbbe2b80)